packages/components/addon/components/hds/form/masked-input/base.hbs

```diff
@@ -2,33 +2,38 @@
   Copyright (c) HashiCorp, Inc.
   SPDX-License-Identifier: MPL-2.0
 }}
-<div class={{this.classNames}} {{style width=@width}}>
-  {{#if @isMultiline}}
-    <Hds::Form::Textarea::Base
-      class="hds-form-masked-input__control"
-      @value={{@value}}
-      @isInvalid={{@isInvalid}}
-      @height={{@height}}
-      id={{this.id}}
-      ...attributes
-    />
-  {{else}}
-    <Hds::Form::TextInput::Base
-      class="hds-form-masked-input__control"
-      @value={{@value}}
-      @isInvalid={{@isInvalid}}
-      id={{this.id}}
-      ...attributes
-    />
+<div class="hds-form-masked-input__wrapper">
+  <div class={{this.classNames}} {{style width=@width}}>
```

Comment:
One thing I noticed with this implementation is that now the definition of what `@width` applies to is ambiguous: if as a consumer I say `@width="200px"` am I including the `CopyButton` or not? Not obvious 🤔